src/utils_request.js

```diff
@@ -254,7 +255,7 @@ function fixUrl(url) {
         } while (url[index] === '%');
 
         if (chunks.length) {
-            const encoded = encodeURI(Buffer.from(chunks, 'hex').toString());
+            const encoded = encodeURIComponent(Buffer.from(chunks, 'hex').toString());
```

Comment:
`encodeURI` doesn't encode `/`: `%2F (input) → / (hex) → / (encodeURI)`
`encodeURIComponent` does encode `/`: `%2F (input) → / (hex) → %2F (encodeURIComponent)`

We want the latter since we deal with percent encoded characters.